projects/ion/src/lib/icon/svgs/icons.ts

```diff
@@ -1,21 +1,56 @@
+import {
+  access,
+  add,
+  check,
+  checkSolid,
+  clockSolid,
+  close,
+  closeSolid,
+  down2,
```

Comment:
could be doubleDown

Reply:
I'm following figma nomenclature. Wouldn't modifying cause any problems?

Reply:
![image](https://user-images.githubusercontent.com/15896994/187432008-f1fec862-78b6-4217-be45-389162a27ec5.png)

Reply:
![image](https://user-images.githubusercontent.com/15896994/187432115-0fceda6f-144e-4a90-b147-8933970abc61.png)

Reply:
Hi @jairochaves2! I don't think left2, down2, etc., are good names. I think they could be more suggestive. We could talk about it with Edney, I don't think he intended to leave the names like that in Figma.

Reply:
Hi, @mizek1. I removed the down2 and up2, and kept the semi-down and semi-up.

Reply:
Was necessary to maintain compatibility.